source/user-manual/reference/ossec-conf/localfile.rst

```diff
@@ -595,6 +597,96 @@ For example, we may want to read a Python Traceback output as one single log, re
       <multiline_regex replace="wspace">^Traceback</multiline_regex>
    </localfile>
 
+ignore
+^^^^^^^^^^^^^^^
+
+This allows you to configure a pcre2 expression to ignore specific log lines. Multiple expressions can be configured by repeating the ignore label. The log will be ignored as soon as one of the expressions matches, OR behavior.
+
++--------------------+---------------------------------------------------------------+
+| **Default Value**  | n/a                                                           |
++--------------------+---------------------------------------------------------------+
+| **Allowed values** | Any `regex <regex.html#regex-os-regex-syntax>`_,              |
+|                    | `sregex <regex.html#sregex-os-match-syntax>`_ or              |
+|                    | `pcre2 <regex.html#pcre2-syntax>`_ expression.                |
++--------------------+---------------------------------------------------------------+
+
+The attribute type is optional.
+
++-------------+-----------------------------------------+-------------+---------------+
```

Comment:
Use this table format:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/29475387/193076056-4eea389a-0ce8-45ee-a7e9-501c7059f462.png">